src/AutoRest.CSharp/Common/Output/Models/Shared/Parameter.cs

```diff
@@ -97,7 +97,7 @@ public static Parameter FromInputParameter(in InputParameter operationParameter,
         public static FormattableString CreateDescription(InputParameter operationParameter, CSharpType type, IEnumerable<string>? values, Constant? defaultValue = null)
         {
             FormattableString description = string.IsNullOrWhiteSpace(operationParameter.Description)
-                ? (FormattableString)$"The {operationParameter.Type.Name} to use."
+                ? (FormattableString)$"The {TypeFactory.GetInputType(type).ToStringForDocs()} to use."
```

Comment:
Could we assume that the type here is **exactly** the type of the parameter which we are creating descriptions on?
I believe we should ensure that - and then we do not need the `TypeFactory.GetInputType` here because not every time when we create the description, it represents the "input" type.

Reply:
Same question why didn't we just use `this.Type`?